contracts/oxo/OxoGame.sol

```diff
@@ -0,0 +1,137 @@
+pragma solidity 0.4.19;
+
+import "./OxoConfig.sol";
+import "./Players.sol";
+import "./GameAction.sol";
+import "./Ownable.sol";
+
+/*In this game one user create a game and set how much he wants to bet.
+Another user should join the game and send equivalent bet number.
+Then game starts*/
+
+contract OxoGame is OxoConfig, GameEntity, GameAction, Players, Ownable {
+
+    event GameCreated(uint gameId, address creator, uint bet);
+    event GameStarted(uint gameId, address player);
+    event UserMadeMove(uint gameId, address player, uint8 index);
+    event GameWon(uint gameId, address winner);
+    event Draw(uint gameId);
+    event MoneyTransferred(uint gameId, address player, uint amount);
+    event GameCanceled(uint gameId);
+    event MoneyWithdrawn(address addr, uint amount);
+
+
+    function createGame() payable public returns (uint){
+        require(msg.value > 0 && msg.value <= maxBet);
+
+
+        Game storage game = createAndGetGame();
+        game.status = GameStatus.CREATED;
+        game.bet = msg.value;
+        game.gameId = games.length - 1;
+        addPlayer(game, msg.sender);
+
+        GameCreated(game.gameId, msg.sender, game.bet);
+        return game.gameId;
+    }
+
+    function joinGame(uint gameId) payable public returns (bool){
+        Game storage game = games[gameId];
+        require(game.status == GameStatus.CREATED);
+        require(msg.value == game.bet);
+        require(game.playerCount < maxPlayerCount);
```

Comment:
no, if the `game.playerCount` equals `maxPlayerCount` then it would be   impossible to join the game.. (all sits are busy )

Reply:
Yeah, I understand. But you could change number of maxPlayerCount. 
Like first player count = 0, second player count = 1. 
maxPlayerCount = 1.
So when you write require(game.playerCount == maxPlayerCount)
it would be
1. more intuitive
2. will work in a right way

But it doesn't necessary. Just advice
Because it's logical when you have maximal count of players and then can play. You can't have more than maximum players.

Reply:
i complicatedly do not understand why it  should be more intuitive  ?
`require(game.playerCount < maxPlayerCount);` says - u can join to the game if there are not enough players. why it is not intuitive ?
`require(game.playerCount == maxPlayerCount)` says -  u can join  to the game if all players are already been joined... right ?  so why it is better